CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v0.17.0
+
+## Major changes
+- New API for specifying sampling methods (#487). Old syntax:
+```python
+posterior = inference.build_posterior(sample_with_mcmc=True)
+```
+New syntax:
+```python
+posterior = inference.build_posterior(sample_with="mcmc")  # or "rejection"
+```
+- Rejection sampling for likelihood(-ratio)-based posteriors (#487)
+- MCMC in unconstrained and z-scored space (#510)
+
+## Minor changes
+- `scatter` allowed for diagonal entries in pairplot (#510)
+- Changes to default hyperparameters for `SNPE_A` (thanks @famura, #496, $497)
+- bugfix for `within_prior` checks (#506)
```

Comment:
update this to a major change. the prior has to be on the same device as passed for training. Nothing changes in the default case, but when passing `device=cuda` and passing a prior as before one will get an `AssertionError`. Instead one has to initialise the prior on the `device` before passing it to SBI.